README.md

```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
```

Comment:
Search for `version=` in the file to find the version number. For example, `1.8.10`.

Reply:
Done.